.pre-commit-config.yaml

```diff
@@ -126,3 +126,9 @@ repos:
     rev: 2025.04.07
     hooks:
       - id: validate-pyproject
+
+  # Check for license headers
+  - repo: https://github.com/emzeat/mz-lictools
+    rev: v2.7.0
+    hooks:
+      - id: license-tools
```

Comment:
I think it might be best to add this _before_ any auto formatters so we can be sure that this check does not produce code that will be reformatted in a subsequent run, which would trigger annoying pre-commit commits in CI.